src/main/java/frc/robot/Robot.java

```diff
@@ -110,19 +141,21 @@ private void configureBindings() {
         manipulator.rightTrigger()
           .whileTrue(algae.setWheelAction(Algae.IntakeSpeed.PROCESSOR_SHOOT));
         manipulator.leftTrigger().and(manipulator.b())
-          .whileTrue(coral.intake());
+          .whileTrue(coral.setCoralMotorAction(kCoralSpeed));
         
         // elevator controls
-        manipulator.leftBumper().onTrue(elevator.toHome());
-        manipulator.povDown().onTrue(elevator.setPosition(Elevator.EleHeights.L1));
-        manipulator.povRight().onTrue(elevator.setPosition(Elevator.EleHeights.L2));
-        manipulator.povLeft().onTrue(elevator.setPosition(Elevator.EleHeights.L3));
-        manipulator.povUp().onTrue(elevator.setPosition(Elevator.EleHeights.L4));
-        manipulator.rightStick().whileTrue(elevator.setPosition(manipulator.getRightX())); // might need lambda?
-
+        // todo: leftBumper, home request
+        manipulator.leftBumper().onTrue(elevator.toPosition(EleHeight.HOME));
+        
+        manipulator.povDown().onTrue(superstructure.requestToScore(EleHeight.L1));
+        manipulator.povRight().onTrue(superstructure.requestToScore(EleHeight.L2));
+        manipulator.povLeft().onTrue(superstructure.requestToScore(EleHeight.L3));
+        manipulator.povUp().onTrue(superstructure.requestToScore(EleHeight.L4));
+        
         // climber controls
-        manipulator.a().and(manipulator.povUp()).onTrue(elevator.setPosition(Elevator.EleHeights.CLIMB_UP));
-        manipulator.a().and(manipulator.povDown()).onTrue(elevator.setPosition(Elevator.EleHeights.CLIMB_DOWN));
+        // TODO: ask superstructure if in idle first
```

Comment:
this still isn't done - either force the Superstructure into idle or check that it's at idle